packages/signals_core/lib/src/async/async_signal.dart

```diff
@@ -12,11 +12,12 @@ typedef AsyncSignalErrorBuilder<R> = R Function(Object? error);
 /// Builder on a [AsyncSignal] callback
 typedef AsyncSignalBuilder<R> = R Function();
 
```

Comment:
What's the use of the 3 typedef, I see no refs elsewhere.

Reply:
It was for the map/maybe map but not needed anymore. Will remove since this is v3